dvf_csv/src/Plugin/Visualisation/Source/CsvFile.php

```diff
@@ -239,7 +239,7 @@ public function getData() {
   protected function fetchData() {
     try {
       $uri = $this->config('uri');
-      $response = $this->httpClient->get($uri)->getBody()->getContents();
```

Comment:
@jez500 should the httpClient property then be removed from both CsvFile and JsonFile classes?
It does not seem to be used anywhere else:
https://github.com/govCMS/dvf/blob/966a0c39959692f648e09f2ce491476d2b649a43/dvf_csv/src/Plugin/Visualisation/Source/CsvFile.php#L41

Reply:
Good catch, @thery-dh removed now